src/applications/appeals/995/tests/995-contact-loop.cypress.spec.js

```diff
@@ -114,7 +135,7 @@ describe('995 contact info loop', () => {
   });
 
   // eslint-disable-next-line @department-of-veterans-affairs/axe-check-required
-  it('should edit info on a new page, update & return to contact info page - C31614', () => {
```

Comment:
We should keep this TestRail ID

Reply:
Ah ok in that case don't think the other one is duplicate either